pkg/util/cmapichecker/cmapichecker.go

```diff
@@ -38,12 +38,13 @@ var (
 )
 
 const (
-	crdsMappingError  = `error finding the scope of the object: failed to get restmapping: no matches for kind "Certificate" in group "cert-manager.io"`
+	crdsMapping1Error = `error finding the scope of the object: failed to get restmapping: failed to find API group "cert-manager.io"`
```

Comment:
There was a chanage in controller-runtime that caused this regex to stop matching.